server/controllers/TripController.js

```diff
@@ -304,6 +304,40 @@ class TripController {
       throw new Error('Could not delete trip')
     }
   }
+
+  async enableShareTrip(userId, id) {
+    try {
+      await TripModel.updateOne(
```

Comment:
The good practice will be for enableShareTrip to call updateTrip controller function

Reply:
But our updateTrip controller function is kinda doomed rn :((,so u can leave here if u dont wanna refactor